sql/zone_settings.sql

```diff
@@ -275,8 +275,8 @@ INSERT INTO `zone_settings` VALUES (230,1,'127.0.0.1',54230,'Southern_San_dOria'
 INSERT INTO `zone_settings` VALUES (231,1,'127.0.0.1',54230,'Northern_San_dOria',107,107,107,107,0,0.00,5192);
 INSERT INTO `zone_settings` VALUES (232,1,'127.0.0.1',54230,'Port_San_dOria',107,107,107,107,0,0.00,5704);
 INSERT INTO `zone_settings` VALUES (233,1,'127.0.0.1',54230,'Chateau_dOraguille',156,156,156,156,0,0.00,5128);
-INSERT INTO `zone_settings` VALUES (234,1,'127.0.0.1',54230,'Bastok_Mines',152,152,152,152,0,0.00,4680);
-INSERT INTO `zone_settings` VALUES (235,1,'127.0.0.1',54230,'Bastok_Markets',152,152,152,152,0,0.00,4680);
+INSERT INTO `zone_settings` VALUES (234,1,'127.0.0.1',54230,'Bastok_Mines',152,152,152,152,0,0.00,5704);
```

Comment:
5704 = 0x1648 which = 
0x0008 (MISC_MAZURKA, good)
0x0040 (MISC_COSTUME, good)
0x0200 (MISC_AH, good)
0x0400 (what the or was for, MISC_YELL, obv good)
and.. 0x1000, (MISC_LOS_PLAYER_BLOCK.. wait what?)

`Players can't use magic/JAs through walls if this is set`

That...that's a thing in town? someone was legit concerned you could cure in town through a door or something??

Well ok then /shrug